app/src/main/java/com/shagalalab/constitution/article/adapter/ArticleAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package com.shagalalab.constitution.article.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.shagalalab.constitution.R
+import com.shagalalab.constitution.data.models.ArticleModel
+
+class ArticleAdapter : RecyclerView.Adapter<ArticleItemViewHolder>() {
+
+    private var models: List<ArticleModel> = arrayListOf()
+
+    fun setData(models: List<ArticleModel>) {
+        this.models = models
+        notifyDataSetChanged()
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ArticleItemViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.item_view, parent, false)
```

Comment:
Using `R.layout.item_view` here is wrong. First of all, you are only using only of the textviews here, which means you need to create a separate layout for it. Do not generalize the layout, if there is something different, then create a new one. Reusing should be only done when most of the views inside are used.